src/EncryptDecryptString.dfy

```diff
@@ -68,8 +68,8 @@ method encrypt_string(key_arn: string,
 
   var plaintext_consumed;
   r, ciphertext_len, plaintext_consumed := session.Process(
-    ciphertext, ciphertext.Length,
-    plaintext, plaintext_len);
+                                             ciphertext, ciphertext.Length,
+                                             plaintext, plaintext_len);
```

Comment:
Ok, I'll stop, but not a fan

Reply:
Here we should add a newline after `:=` if you don't want all this whitespace.
I understand how having only 2 indent spaces after "var" saves whitespace, but I don't think the previous way is  more legible. Arguments were too far visually from the function. Now, ok, there is whitespace, but arguments are next to the function in a format very easy to recognize.